src/main/java/org/openstreetmap/atlas/checks/flag/FlaggedPoint.java

```diff
@@ -25,6 +24,7 @@
  */
 public class FlaggedPoint extends FlaggedObject
 {
+    protected static final String SYNTHETIC_POINT_TAG = "synthetic_highlight_point";
```

Comment:
It would be useful to have this as a tag enum, so other classes could access it (such as the diff commands).

Reply:
In atlas ?

Reply:
As this is very checks specific I think it might make more sense to have it in the Atlas Checks project, though in the same vein as the tags in Atlas. Not sure exactly the best place for it to go in Atlas Check, I would suggest in the flag, constants, or utility packages.

Reply:
Created a utility.tags package & added this as an enum